### PR TITLE
refactor: updated versions for release

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**

### CODE REVIEW

File: packages/contentstack-command/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "format": "eslint src/**/*.ts --fix"
   },
   "dependencies": {
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```
Comment (antonyagustine) [inline packages/contentstack-command/package.json:20]: @abhinav-from-contentstack Update the package version.

And need to update all the location where this package is used.

CC @netrajpatel

**Review by antonyagustine — CHANGES_REQUESTED** [review]

File: packages/contentstack-migration/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@contentstack/cli-command": "^1.0.1",
     "@contentstack/management": "^1.3.0",
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```
Comment (antonyagustine) [inline packages/contentstack-migration/package.json:9]: We need to update this package version and all location it is used

File: packages/contentstack-migrate-rte/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "^1.0.1",
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```
Comment (antonyagustine) [inline packages/contentstack-migrate-rte/package.json:9]: We need to update this package version and all location where this package is used

**Review by antonyagustine — CHANGES_REQUESTED** [review]

File: packages/contentstack-migration/package.json (hunk 2 above)
Comment (abhinav-from-contentstack) [reply packages/contentstack-migration/package.json:9]: done

File: packages/contentstack-migrate-rte/package.json (hunk 3 above)
Comment (abhinav-from-contentstack) [reply packages/contentstack-migrate-rte/package.json:9]: done

File: packages/contentstack-command/package.json (hunk 1 above)
Comment (abhinav-from-contentstack) [reply packages/contentstack-command/package.json:20]: done

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**